### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by iakat**

### CODE REVIEW

File: _posts/2000-01-03-details.md [hunk 1]
```diff
@@ -35,4 +35,4 @@ If the installation doesn't work for you, check out [common errors documentation
 
 ### Simply join our channel at [Gitter](https://coala.io/chat)! We'd love to speak to you!
 
-### Look at [our tutorial](https://coala.readthedocs.io/en/latest/Users/Tutorials/Tutorial.html) for a deeper introduction.
+### Look at [our tutorial](https://coala.io/tutorial) for a deeper introduction.
```
Comment (sils) [inline _posts/2000-01-03-details.md:38]: getting a 404 here

Comment (iakat) [reply _posts/2000-01-03-details.md:38]: solved

Comment (sils) [reply _posts/2000-01-03-details.md:38]: maze found, redirect to https://docs.coala.io/en/latest/Users/Tutorial.html instead. The link was apparently even outdated.

File: _posts/2000-01-08-getinvolved.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ color: white
...
 ---
 
-# Take a look at our [newcomer docs](http://coala.readthedocs.io/en/latest/Developers/Newcomers_Guide.html)!
+# Take a look at our [newcomer docs](https://coala.io/newcomer)!
 
-### Or just drop us a message at [Gitter](https://gitter.im/coala-analyzer/coala) - we'd love to help you personally!
+### Or just drop us a message at [Gitter](https://coala.io/chat) - we'd love to help you personally!
 
-### If you're looking to write a bear, look at our [bear writing tutorial](http://coala.readthedocs.io/en/latest/Developers/Writing_Bears.html).
+### If you're looking to write a bear, look at our [bear writing tutorial](https://docs.coala.io/writingbears).
```
Comment (sils) [inline _posts/2000-01-08-getinvolved.md]: wrong link? Was this meant to be coala.io/writingbears ?

File: _posts/2000-01-03-details.md (hunk 1 above)
Comment (iakat) [reply _posts/2000-01-03-details.md:38]: :joy_cat:

Comment (sils) [reply _posts/2000-01-03-details.md:38]: still redirects to maze

Comment (iakat) [reply _posts/2000-01-03-details.md:38]: Caching hates you.

**Comment by sils** [comment]
Thanks!

File: _posts/2000-01-03-details.md (hunk 1 above)
Comment (abhigyank) [inline _posts/2000-01-03-details.md]: @chauffer @sils how is the redirection being done?
Where do I get that https://coala.io/tutorial is redirecting to https://docs.coala.io/en/latest/Users/Tutorial.html ?

**Comment by sils** [comment]
check coala.io/links

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Pull request merged**